src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,140 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+
+    private class Node<T> {
```

Comment:
let's move class declaration to the end of the file since it has private modifier

Reply:
done

Reply:
done